app/packages/core/src/components/Schema/SchemaSettings.tsx

```diff
@@ -0,0 +1,235 @@
+import React, { Fragment, useEffect, useRef } from "react";
+import styled from "styled-components";
+
+import * as fos from "@fiftyone/state";
+
+import CloseIcon from "@mui/icons-material/Close";
+import { Box, Typography } from "@mui/material";
+
+import { Button, useTheme } from "@fiftyone/components";
+import { TabOption } from "../utils";
+
+import useSchemaSettings, {
+  TAB_OPTIONS,
+  TAB_OPTIONS_MAP,
+} from "@fiftyone/state/src/hooks/useSchemaSettings";
+
+import { SchemaSearch } from "./SchemaSearch";
+import { SchemaSelection } from "./SchemaSelection";
+import { graphql } from "react-relay";
```

Comment:
nit: unused. I've been using the sort imports flag in vscode formatting which helps for consistency too
```
  "[typescript]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode",
    "editor.codeActionsOnSave": {
      "source.fixAll.eslint": true,
      "source.organizeImports": true,
      "source.sortMembers": true
    },
    "editor.formatOnSave": true, 
    "editor.formatOnPaste": false, 
  },
  "[typescriptreact]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode",
    "editor.codeActionsOnSave": {
      "source.fixAll.eslint": true,
      "source.organizeImports": true,
      "source.sortMembers": true
    },
    "editor.formatOnSave": true, 
    "editor.formatOnPaste": false, 
  }
```